storage/range.go

```diff
@@ -235,13 +235,35 @@ func (r *Range) String() string {
 	return fmt.Sprintf("range=%d (%s-%s)", r.Desc().RaftID, r.Desc().StartKey, r.Desc().EndKey)
 }
 
-// start begins by gossiping the cluster ID (only happens if the range
-// is the first in the key space). This is necessary to get the cluster
-// started from scratch.
+// start begins by gossiping the configs. It also gossips the sentinel if we're
+// the first range (the store will do this regularly, but this is a good time
+// to get started quickly).
 func (r *Range) start() {
-	if r.IsFirstRange() {
-		r.startGossip(r.rm.Stopper())
+	// Being the first range comes with extra responsibility for the gossip
```

Comment:
Feels to me like we should find some way to get rid of `Range.start()` completely.

The biggest reason is that "start" as a verb usually means you're starting a goroutine. That's not the case anymore, so let's eliminate this as it's confusing. Second, the logic here is tortured. Feels like it's not the way you'd write it, except you've been wrestling with byzantine test failures. I know the feeling.